src/main/java/frc/robot/subsystems/gpm/Shooter.java

```diff
@@ -43,239 +44,244 @@ public class Shooter extends SubsystemBase {
-	
-    // TODO: TUNE THIS
-    private final SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(S, V);
-	
-    public Shooter() {
+	private final PIDController rightPID = new PIDController(P, I, D);
+	private FlywheelSim rightFlywheelSim;
+	private double rightMotorSpeedSim;
+	private double rightPower = 0.0;
+
+	// TODO: TUNE THIS
+	private final SimpleMotorFeedforward feedforward = new SimpleMotorFeedforward(S, V);
+
+	public Shooter() {
 		// set the RPM tolerance of the PID controllers
-        topPID.setTolerance(TOLERANCE);
-        bottomPID.setTolerance(TOLERANCE);
+		leftPID.setTolerance(TOLERANCE);
+		rightPID.setTolerance(TOLERANCE);
 		// invert the bottom motor so +power sends the note out
-        bottomMotor.setInverted(true);
+		rightMotor.setInverted(true);
 
 		// are we simulating?
 		if (RobotBase.isSimulation()) {
-			topFlywheelSim = new FlywheelSim(gearbox, 1.0, MOI_SHAFT);
-			bottomFlywheelSim = new FlywheelSim(gearbox, 1.0, MOI_SHAFT);
+			leftFlywheelSim = new FlywheelSim(gearbox, 1.0, MOI_SHAFT);
+			rightFlywheelSim = new FlywheelSim(gearbox, 1.0, MOI_SHAFT);
 		}
-    }
+	}
 
-    @Override
-    public void periodic() {
+	@Override
+	public void periodic() {
 		// PID loop uses RPM
-        double topSpeed = getTopMotorRPM();
-        double bottomSpeed = getBottomMotorRPM();
+		double leftSpeed = getTopMotorRPM();
+		double rightSpeed = getBottomMotorRPM();
 
-		// TODO: having problems: a set and get do not match, so keep powers around for simulation
-		topPower = topPID.calculate(topSpeed) + feedforward.calculate(topPID.getSetpoint());
-		bottomPower = bottomPID.calculate(bottomSpeed) + feedforward.calculate(bottomPID.getSetpoint());
+		// TODO: having problems: a set and get do not match, so keep powers around for
+		// simulation
+		leftPower = leftPID.calculate(leftSpeed) + feedforward.calculate(leftPID.getSetpoint());
+		rightPower = rightPID.calculate(rightSpeed) + feedforward.calculate(rightPID.getSetpoint());
 
 		// set motor powers
-        topMotor.set(topPower);
-        bottomMotor.set(bottomPower);
+		leftMotor.set(leftPower);
+		rightMotor.set(rightPower);
 
 		// report some values to the Dashboard
-        SmartDashboard.putNumber("top speed", /* shooterRPMToSpeed */ (topSpeed));
-        SmartDashboard.putNumber("bottom speed", /* shooterRPMToSpeed */ (bottomSpeed));
-        SmartDashboard.putBoolean("at setpoint?", atSetpoint());
-    }
+		SmartDashboard.putNumber("top speed", /* shooterRPMToSpeed */ (leftSpeed));
+		SmartDashboard.putNumber("bottom speed", /* shooterRPMToSpeed */ (rightSpeed));
+		SmartDashboard.putBoolean("at setpoint?", atSetpoint());
+	}
 
 	@Override
 	public void simulationPeriodic() {
 		// assume the battery voltage is 12 volts
 		double voltage = 12.0;
 
-		// topPower and bottomPower are now member variables; cannot read them from simulated encoder
+		// topPower and bottomPower are now member variables; cannot read them from
```

Comment:
Again, change comment. Also remove commented out code.

Reply:
commented out code is so aesthetic though!
(it's removed)